src/h3/api/basic_int/__init__.py

```diff
@@ -996,3 +996,70 @@ def great_circle_distance(latlng1, latlng2, unit='km'):
         lat2, lng2,
         unit = unit
     )
+
+
+def cell_to_vertex(h, vertex_num):
+    """
+    Return a (specified) vertex of an H3 cell.
+
+    Parameters
+    ----------
+    h : H3Cell
+    vertex_num : int
+        Vertex number (0-5)
+
+    Returns
+    -------
+    The vertex
+    """
+    return _cy.cell_to_vertex(_in_scalar(h), vertex_num)
+
+
+def cell_to_vertexes(h):
+    """
+    Return a list of vertices of an H3 cell. The list will always be of length 6.
+    If a pentagon is entered, the last element will be 0.
+
+    Parameters
+    ----------
+    h : H3Cell
+
+    Returns
+    -------
+    A list of vertices
+    """
+    mv = _cy.cell_to_vertexes(_in_scalar(h))
+    arr = [str_to_int(a) for a in _out_collection(mv)]
+    return arr if len(arr) == 6 else arr + [0]
+
+
+def vertex_to_latlng(v):
+    """
+    Return latitude and longitude of a vertex.
+
+    Returns
+    -------
+    lat : float
+        Latitude
+    lng : float
+        Longitude
+    """
+    if isinstance(v, str):
+        v = str_to_int(v)
+    return _cy.vertex_to_latlng(v)
```

Comment:
You need to use `_in_scalar()` so that this function will work as expected for the other APIs (`basic_int`, `basic_str`, etc.).

Reply:
OK!